drivers/volume/portworx/portworx.go

```diff
@@ -980,6 +1088,25 @@ func (d *portworx) pickAlternateClusterManager(n node.Node) (api.OpenStorageNode
 	return nil, fmt.Errorf("failed to get an alternate cluster manager for %s", n.Name)
 }
 
+// EnableStorageExpansion creates autopilot rules
+func (d *portworx) EnableStorageExpansion(apRules []apapi.AutopilotRule) error {
```

Comment:
The method says `EnableStorageExpansion` but in reality it isn't doing that. 

The implementation just takes in a bunch of apRules and applies them. There is an implicit assumption that the caller has written rules that perform storage expansion.